content/en-us/learn/tokenomics/tokenomics.md

```diff
@@ -143,27 +146,30 @@ $$\Delta V_t = k_p \cdot \big((\rho^m - 1) V_t + c(s_t) + \gamma(V_t)h(V_t)\big)
 
 Proposed parameters:
 
-- $\rho^m_{\text{Khala}} = 1.00020$ (hourly)
-- $\rho^m_{\text{Phala}} = 1.00020$ (hourly)
+- $\rho^m_{\text{Phala}} =\rho^m_{\text{Khala}} = 1.00020$ (hourly)
 
 ### Payout Event
 
-In order to stay within the subsidy budget, at every block the budget is distributed proportionally based on the current **_Miner Shares_**. However, the payout is also capped to ensure the payout doesn't cause $V$ to drop lower than it in the last payout event:
-
-$$w(V_t) = \min(B \frac{\text{share}}{\Sigma \text{share}}, V_t - V_\text{last}),$$
+In order to stay within the subsidy budget, at every block the budget is distributed proportionally based on the current **_Miner Shares_**:
 
-where $B$ is the current network subsidy budget for the given payout period, and $V_\text{last}$ is the value promised at the last payout event, or $V^e$ if this is the first payout.
+$$w(V_t) = B \frac{\text{share}}{\Sigma \text{share}}$$
 
-> Capping the payout is necessary to make sure miners are well incentives to always accumulate credits in the network. Otherwise if V decreases over the time, miners are incentivezed to constantly reset their mining session.
+where $B$ is the current network subsidy budget for the given payout period.
 
 Whenever $w(V_t)$ is paid to a miner, his $V$ will be updated accordingly:
 
-$$\Delta V = -w(V_t).$$
+$$\Delta V = -min(w(V_t),V_t-V_\text{last}).$$
+
+$V_\text{last}$ is the value promised at the last payout event, or $V^e$ if this is the first payout.
+
+>The update of V is limited to ensure the payout doesn't cause $V$ to drop lower than it was in the last payout event. The limit is necessary to make sure miners are well incentives to always accumulate credits in the network. Otherwise, miners are incentivized to constantly reset their mining session if V decreases over time.
```

Comment:
```suggestion
> The update of $V$ is limited to ensure the payout doesn't cause $V$ to drop lower than it was in the last payout event. The limit is necessary to make sure miners are well incentives to always accumulate credits in the network. Otherwise, miners are incentivized to constantly reset their mining session if V decreases over time.
```